### PR TITLE
[Feat]:Naviagtion component added

### PR DESCRIPTION
**Pull request by t007rushi**
Responsive navigation component added with mobile-first approach
Hiding & visible content Nav provided with basic important buttons

### CODE REVIEW

**Review by Manju-sree — APPROVED** [review]
code looks good to merge

**Review by shaksham08 — APPROVED** [review]
Looks good to me

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+.cart {
+  font-size: 1rem;
+  cursor: pointer;
+  gap: 0.5rem;
+  align-items: center;
+}
+
+.search-bar:focus {
+  outline: none;
```
Comment (dev-enforced) [inline components/navigation.css]: Instead of outline:none you can give outline:2px solid transparent; so that can avoid accessibility issues

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!! Will suggest the use of utility classes for optimising the tasks

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  width: 100%;
+  border: none;
+}
+
+.header-right {
+  width: 22rem;
+  align-items: center;
+  justify-content: space-between;
+  gap: 2rem;
+  width: 30%;
```
Comment (kotalwar-krishna) [inline components/navigation.css:41]: there is any purpose for using width in %. elsewhere you used rem unit for width only where width 100% required that time you use %.

**Review by kotalwar-krishna — APPROVED** [review]
Good to merge

File: components/navigation.css (hunk 2 above)
Comment (t007rushi) [reply components/navigation.css:41]: yes for making responsivness as it gets minimized according to page size

**Pull request merged**